modules/serverless-installing-cli-linux.adoc

```diff
@@ -0,0 +1,50 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative-client.adoc
+
+[id="installing-cli-linux_{context}"]
+= Installing `kn` for Linux
+
+For Red Hat Enterprise Linux (RHEL), you can install `kn` as an RPM if you have an active OpenShift Enterprise subscription on your Red Hat account:
```

Comment:
"OpenShift Enterprise" should use the {product-title} attribute so that it correctly replaces with the product name (OpenShift Container Platform). Same for the other instance in this file.